parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileWriter.java

```diff
@@ -558,6 +555,14 @@ private void innerWriteDataPage(
     currentEncodings.add(valuesEncoding);
   }
 
+  /**
+   * Write a Bloom filter
+   * @param bloomFilter the bloom filter of column values
+   */
+  public void writeBloomFilter(BloomFilter bloomFilter)  {
```

Comment:
Currently, only `TestParquetFileWriter` uses this method. If it is not really required to be `public` I would suggest having it package protected.

Reply:
Thanks, Done.

Reply:
I've ment _package protected_ (aka _package private_) which is marked with using no modifiers. 
```suggestion
  void writeBloomFilter(BloomFilter bloomFilter)  {
```

Reply:
Oops, understood now.